torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -23,16 +21,16 @@
 
 
 class Raft_Large_Weights(WeightsEnum):
-    pass
-    # C_T_V1 = Weights(
-    #     # Chairs + Things
-    #     url="",
-    #     transforms=RaftEval,
-    #     meta={
-    #         "recipe": "",
-    #         "epe": -1234,
-    #     },
-    # )
+    C_T_V2 = Weights(
+        # Chairs + Things
+        url="https://download.pytorch.org/models/raft_large_C_T_V2-1bb1363a.pth",
+        transforms=RaftEval,
+        meta={
+            "recipe": "",  # TODO
```

Comment:
You can add the URL of the `references/optical_flow/README.md` file here as if it was on main branch.